sdks/python/container/boot.go

```diff
@@ -170,6 +176,28 @@ func main() {
 	log.Fatalf("Python exited: %v", execx.Execute("python", args...))
 }
 
+// setup wheel specs according to installed python version
+func setupAcceptableWheelSpecs() error {
+	cmd := exec.Command("python", "-V")
+	stdoutStderr, err := cmd.CombinedOutput()
+	if err != nil {
+		return err
+	}
+	re := regexp.MustCompile(`Python (\d)\.(\d).*`)
+	pyVersions := re.FindStringSubmatch(string(stdoutStderr[:]))
+	if len(pyVersions) != 3 {
+		return fmt.Errorf("cannot get parse Python version from %s", stdoutStderr)
+	}
+	pyVersion := fmt.Sprintf("%s%s", pyVersions[1], pyVersions[2])
+	if pyVersion == "27" {
+		acceptableWhlSpecs = append(acceptableWhlSpecs, "cp27-cp27mu-manylinux1_x86_64.whl")
+	} else {
+		wheelName := fmt.Sprintf("cp%s-cp%sm-manylinux1_x86_64.whl", pyVersion, pyVersion)
```

Comment:
Sorry for a slow response. It seems that 3.8 wheels don't follow this pattern, just noticing this while prepring 2.23.0 release. 3.8 wheels look as follows:

apache_beam-2.23.0-cp38-cp38-manylinux1_x86_64.whl

Not sure why the pattern has changed.

Reply:
Thank you for notification.
I found that wheel changed naming rule at 0.33.5.
> 0.33.5 (2019-08-17)
> Don’t add the m ABI flag to wheel names on Python 3.8 (PR by rdb)
https://wheel.readthedocs.io/en/stable/news.html

I'll push a fix tonight, wait a bit.

Reply:
Thanks! Also filed https://issues.apache.org/jira/browse/BEAM-10434 to fix the stager.